be/src/io/cache_input_stream.cpp

```diff
@@ -64,8 +64,13 @@ CacheInputStream::CacheInputStream(const std::shared_ptr<SharedBufferedInputStre
 
 CacheInputStream::~CacheInputStream() {
     int64_t io_bytes = _sb_stream->shared_io_bytes();
```

Comment:
why not consider both direct and shared io bytes?

Reply:
> why not consider both direct and shared io bytes?

Yes, at first, I felt that direct io was often splited, which may make the estimation inaccurate. However, in some cases, it may be more appropriate to consider it. I have changed.